BlueShift-iOS-SDK/BlueShiftDeviceData.m

```diff
@@ -30,7 +30,10 @@ - (NSString *)deviceUUID {
             [defaults setObject:UUID forKey: @"BlueshiftDeviceIdSourceUUID"];
             deviceUUID = [UUID copy];
         }
-    } else {
+    } else if (_blueshiftDeviceIdSource && _blueshiftDeviceIdSource == BlueshiftDeviceIdSourceIDFVBundleID && [[NSBundle mainBundle] bundleIdentifier] != nil) {
```

Comment:
I prefer moving the bundleIdentifier nil check inside the this if block and then log a error if it is nil. this will help for debugging

Reply:
Resolved